### DISCUSSION TITLE
Is the application supposed to change pattern selection methods everytime it starts up?

### DISCUSSION THREAD

**Discussion by lseifert** [hydrogen-music/hydrogen#1540] [General]
Someone help me out is the correct way of using the software to run it on a system that can not be used with an identifiable user and to hide in ones basement?  Or is there a reason that this software changes its patterns select method every time I boot it?  That a feature or just a funny little gag to help users select different software?

**Comment by theGreatWhiteShark** [comment]
>  is there a reason that this software changes its patterns select method every time I boot it

Nope. This is a minor bug. It will be fixed in the next release. Thanks for reporting.
